test/spark/run-test.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh -e
+#
+# run-test.sh - submit spark app after setup lakefs and upload dataset.
+#
+# uses the following environment variables:
+#   STORAGE_NAMESPACE - storange namespace used for repository creation
+# 
+# NOTE that this script should be run from the root project in order for docker compose to volume mount the project
+
+TESTER_ACCESS_KEY_ID=AKIAIOSFODNN7EXAMPLE
+TESTER_SECRET_ACCESS_KEY=wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
+STORAGE_NAMESPACE=${STORAGE_NAMESPACE:-local://}
+
+echo "==> wait for lakefs"
+docker-compose exec -T lakefs /app/wait-for localhost:8000
```

Comment:
Suggestion: use the health-check api instead.

Reply:
I'll defer it to a feature in lakectl or improve wait-for to perform HTTP request